src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs

```diff
@@ -13,6 +13,7 @@ public class IDispatchImplAttributeTests
 
         [Theory]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/50717", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/50717", typeof(PlatformDetection), nameof(PlatformDetection.IsAndroidAOT))]
```

Comment:
Same, as for TagToTokenTests.cs, if the failure is the same and known for same underlying reason, can the issue be updated to include that it fails on Android AOT too?

Reply:
same errors, added a comment to the issue.